### PR TITLE
Add dependabot configuration

### PR DESCRIPTION
**Pull request by jsf9k**
This pull request adds a configuration file for dependabot.

### CODE REVIEW

**Comment by jsf9k** [comment]
@h-m-f-t, can I make Snyk not required for this repo?  It doesn't seem to kick off for me...

**Comment by h-m-f-t** [comment]
@jsf9k Snyk has been useful in the past, but it does seem to hang more often than not.

**Comment by jsf9k** [comment]
> @jsf9k Snyk has been useful in the past, but it does seem to hang more often than not.

I think that is because with a free account you can only run it so many times per month.  It will still attempt to run if I make it "not required", but it won't stop PRs from being able to be merged.

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
```
Comment (mcdonnnj) [inline .github/dependabot.yml:1]: Do we want to add `package-ecosystem: npm`? Since there is a `package.json` here, this seems like something to add.

**Review by mcdonnnj — CHANGES_REQUESTED** [review]
Nice add. Please see my comment about an additional `package-ecosystem`.

**Comment by h-m-f-t** [comment]
> It will still attempt to run if I make it "not required", but it won't stop PRs from being able to be merged.

Ah, yeah, that'd be a useful change.

File: .github/dependabot.yml (hunk 1 above)
Comment (jsf9k) [reply .github/dependabot.yml:1]: Wise.  Added in commit 4837792.

**Review by mcdonnnj — APPROVED** [review]
Nice, thanks for adding the additional checking.

**Pull request merged**